packages/binding/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@realm/binding",
+  "version": "0.1.0",
+  "private": true,
+  "main": "dist/bundle.cjs.js",
+  "module": "dist/bundle.esm.js",
+  "types": "dist/bundle.d.ts",
+  "description": "Binding exposing Realm Core to the SDK running on Node.js",
+  "scripts": {
+    "test": "tsm --no-warnings tests/test.ts",
+    "compile": "cmake-js compile --debug",
```

Comment:
> when someone does an install

I don't expect any end-user to run the compilation of our Node.js binary as we'll use prebuilt and upload these to S3 for download. As such, I'd suggest we instrument the GHA workflow to produce release instead of debug builds. I suggest we cross that bridge when we get there.